### DISCUSSION TITLE
Circular Progress Indicator?

### DISCUSSION THREAD

**Discussion by tskinn** [minad/org-modern#236] [Q&A] [answered]
In the image in the example, instead of a linear progress bar, there appears to be a circular progress pie chart thing. I don't see any reference to something like that in the code. What am I missing? Any guidance on how to get it? Thanks!

**Comment by minad** [comment, marked as answer]
The circular indicator has been replaced in https://github.com/minad/org-modern/commit/ecb590e24d25e59bf84a63a34c29158c55f457eb.

**Reply by tskinn** [reply]
ohhh. Ok thanks for all the great work!
